### ISSUE TITLE
Document type checking toggle

### ISSUE THREAD

**Issue by DMats** [oceanprotocol/ocean.py#260]
Add a few lines to developers.md unit test section that describes how to enable/disable type checking, for a tradeoff between testing speed and thoroughness.

**Comment by DMats** [comment]
It's worth noting that when I tested it, enabling type checking actually sped up execution compared to when it was turned off.

Quoted from https://github.com/oceanprotocol/ocean.py/pull/254#issue-599794098

>Performance Analysis
I ran three trials in which I compared the total run time of pytest with typechecking turned on and turned off. Strangely, in all three cases, the tests executed faster with typechecking enabled.
  | On | Off
-- | -- | --
Trial 1 | 7:56 | 8:23
Trial 2 | 7:25 | 7:33
Trial 3 | 7:36 | 7:51
